ros/processor/insights_engine_consumer.py

```diff
@@ -272,7 +272,6 @@ def trigger_notification(
 
 if __name__ == "__main__":
     start_http_server(int(METRICS_PORT))
-    cache.init_app(app)
```

Comment:
This was an intentional change. Please refer - https://github.com/RedHatInsights/ros-backend/pull/437/files.